resources/bin/orchestrator-client

```diff
@@ -233,6 +233,10 @@ function filter_keys {
   cat - | jq '.[] | .Key'
 }
 
+function filter_broken_replicas {
+  cat - | jq '.[] | select((.Slave_SQL_Running == false or .Slave_IO_Running == false) and (.LastSQLError != "" or .LastIOError != "")) | [.]'
```

Comment:
`[.]` is a trick that wraps each element in its own array, which is not technically an array of objects, but works well for the rest of filtering.